### PR TITLE
Logic feature

### PR DESCRIPTION
**Pull request by fatymahmed**

### CODE REVIEW

**Comment by shloch** [comment]
### PR under review by SHEY Louis CHIA

File: src/logic/operate.js [hunk 1]
```diff
@@ -0,0 +1,17 @@
...
+
+const operate = (numberOne, numberTwo, operation) => {
+  switch (operation) {
+    case '-':
+      return Big(numberOne) - Big(numberTwo);
+    case '+':
+      return Big(numberOne) + Big(numberTwo);
+    case '/':
+      return Big(numberOne) / Big(numberTwo);
+    case '%':
```
Comment (shloch) [inline src/logic/operate.js:11]: You have to manage division by Zero on the live demo. You know that will cause an error. Take note of that on the next milestone

**Review by shloch — APPROVED** [review]
### Great job, 👏👏

### This is Valid for me, you can merge this Pull Request

Happy coding

File: src/logic/operate.js (hunk 1 above)
Comment (fatymahmed) [reply src/logic/operate.js:11]: Thanks, I'll definitely add this

**Comment by fatymahmed** [comment]
> ### Great job, clapclap
> ### This is Valid for me, you can merge this Pull Request
> 
> Happy coding

Thank you @shloch

**Pull request merged**